packages/mds-rpc-common/tests/index.spec.ts

```diff
@@ -0,0 +1,87 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import test from 'unit.js'
+import { ServiceResult, ServiceError, ServiceClient } from '@mds-core/mds-service-helpers'
+import { RpcServiceDefinition } from '../@types'
+import { RpcRoute } from '../index'
+import { RpcClient, RpcRequest } from '../client'
+import { RpcServer } from '../server'
+
+const TEST_WORD = 'mds-rpc-common'
+
+interface TestService {
+  length: (word?: string) => number
+}
+
+export const TestServiceRpcDefinition: RpcServiceDefinition<TestService> = {
+  length: RpcRoute<TestService['length']>()
+}
+
+const TestServer = RpcServer(
+  TestServiceRpcDefinition,
+  {
+    onStart: async () => undefined,
+    onStop: async () => undefined
+  },
+  {
+    length: async ([word]) =>
+      word && word.length > 0 ? ServiceResult(word.length) : ServiceError({ type: 'NotFoundError', message: 'No Word' })
+  }
+).controller()
+
+describe('Test RPC Client', () => {
+  const TestClient: ServiceClient<TestService> = {
```

Comment:
I don't quite understand the purpose of this `length` property. It seems to me like it stores the service name and returns the length of the name if the service was initialized successfully. Am I right?

Reply:
It’s a silly test client method; length is a method that accepts a string, invokes the rpc method which returns its length.

Reply:
Cool, could you add a comment to that effect, that it's purely arbitrary and there for testing?